go

```diff
@@ -167,6 +173,7 @@ CommandGroup.add_group(
   'Development commands',
   {
     :init => 'Set up the Hub dev environment',
+    :update_components => 'Execute bower-installer to update Bower components. Requires Node.js.',
```

Comment:
Doesn't have to be Bower-specific (e.g. it could later include a minification step or something), but I don't feel strongly one way or another.